src/engine/renderer/tr_image.cpp

```diff
@@ -166,23 +172,59 @@ void R_ListImages_f()
 		{ GL_RG16F, { "RG16F", 4 } },
 		{ GL_RG32F, { "RG32F", 8 } },
 
-		// TODO: find imageDataSize multiplier
-		{ GL_COMPRESSED_RGBA, { "RGBAC", 4 } },
-
-		{ GL_COMPRESSED_RGB_S3TC_DXT1_EXT, { "DXT1", 4 / 8 } },
-		{ GL_COMPRESSED_RGBA_S3TC_DXT1_EXT, { "DXT1a", 4 / 8 } },
-		{ GL_COMPRESSED_RGBA_S3TC_DXT3_EXT, { "DXT3", 4 / 4 } },
-		{ GL_COMPRESSED_RGBA_S3TC_DXT5_EXT, { "DXT5", 4 / 4 } },
-
-		// TODO: find imageDataSize multiplier
-		{ GL_COMPRESSED_RED_RGTC1, { "RGTC1r", 1 } },
-		// TODO: find imageDataSize multiplier
-		{ GL_COMPRESSED_SIGNED_RED_RGTC1, { "RGTC1Sr", 1 } },
-		// TODO: find imageDataSize multiplier
-		{ GL_COMPRESSED_RG_RGTC2, { "RGTC2rg", 2 } },
-		// TODO: find imageDataSize multiplier
-		{ GL_COMPRESSED_SIGNED_RG_RGTC2, { "RGTC2Srg", 2 } },
-
+		// Compressed formats here use imageDataSize multiplier as blocksize
+		
+		/* Generic compressed format that does not have any specified actual format
+		   and implementations can compress it in whatever way
+		   GL4.6 spec (https://registry.khronos.org/OpenGL/specs/gl/glspec46.core.pdf):
+		   8.5. TEXTURE IMAGE SPECIFICATION
+		   "Generic compressed internal formats are never used directly as the internal formats of texture images.
+		   If internalformat is one of the six generic compressed internal formats, its value is replaced by the symbolic constant
+		   for a specific compressed internal format of the GL’s choosing with the same base internal format."
+		   Use 4x4 blocks with 8 bytes per block here as an estimate: */
+		{ GL_COMPRESSED_RGBA, { "RGBAC", 8 } },
+
+		/* https://registry.khronos.org/OpenGL/extensions/EXT/EXT_texture_compression_s3tc.txt
+		   S3TC Compressed Texture Image Formats
+		   "Compressed texture images stored using the S3TC compressed image formats
+		   are represented as a collection of 4x4 texel blocks, where each block
+		   contains 64 or 128 bits of texel data.  The image is encoded as a normal
+		   2D raster image in which each 4x4 block is treated as a single pixel.  If
+		   an S3TC image has a width or height that is not a multiple of four, the
+		   data corresponding to texels outside the image are irrelevant and
+		   undefined.
+
+	  	   When an S3TC image with a width of w, height of h, and block size of
+		   blocksize (8 or 16 bytes) is decoded, the corresponding image size (in
+		   bytes) is: ceil( w / 4 ) * ceil( h / 4 ) * blocksize."
+
+		   All of the following have blocksize of 8 bytes: */
+		{ GL_COMPRESSED_RGB_S3TC_DXT1_EXT, { "DXT1", 8 } },
+		{ GL_COMPRESSED_RGBA_S3TC_DXT1_EXT, { "DXT1a", 8 } },
+		{ GL_COMPRESSED_RGBA_S3TC_DXT3_EXT, { "DXT3", 8 } },
+		{ GL_COMPRESSED_RGBA_S3TC_DXT5_EXT, { "DXT5", 8 } },
+
+		/*
```

Comment:
Comment open/end token better be on same line (unless it is ambiguous with the meaning of the line, which is very rare.

Reply:
Yep, missed that one when I was going through the formatting lol.

Reply:
Fixed.